### ISSUE TITLE
lockdown browser issue

### ISSUE THREAD

**Issue by alu1al** [Batlez/HiddenVM#7]
**Hi , first of all you are amazing , thanks for this amazing VM** 

I'm having trouble using your VM. When I install and run Lockdown inside the VM, the installation finishes without problems. The Lockdown browser start page runs fine, but when I try to take a test, I get kicked out right away and see a warning message.

I think the problem might be because I haven't done anything with the antiOS.exe program that comes with it. I haven't opened it and don't know how it works.

Do I need to do something with antiOS.exe before running the Lockdown browser?

![Screenshot 2024-05-30 202838](https://github.com/Batlez/HiddenVM/assets/154839574/9a46ff50-0c1e-44a6-975e-b7d874e73612)

what should i do here to bypass lockdown detection??

Thank you.

**Comment by Batlez** [comment]
Yes you should run ANTIOS before using LockDown Browser. Basically you would change everything in SID Changer, Windows IDs, and License IDs at the top.

**Comment by alu1al** [comment]
I tried, but it didn’t work

the lockdown browser works fine when trying to open it, but when starting the exam, if I switch to the main machine, the browser detects this and kicks me out of the exam.

need your help please

**Comment by alu1al** [comment]
I tried, but it didn’t work. I think the reason is that the Lockdown Browser performs a Pafish check to ensure that the mouse is present on the exam page when starting and during the exam. When it doesn't find the mouse, it detects that the browser is running in a virtual environment (I think this is the reason for the detection, but I am not sure about it). After that, the browser kicks me out of the exam

need your help please

**Comment by Batlez** [comment]
Your mouse should always stay in the virtual machine itself. When you run the RTSC.bat, it minimics the mouse when you leave the virtual machine, so while it's not actually there, it stays and appears there, so it should work with LockDown Browser. I tried it and everything worked fine.

**Comment by sk009900** [comment]
@Batlez Hi! Thank you for posting this VM. I'm a major noob to the tech world and having trouble with literally everything from start to finish, are you able to pls post a video as if ur explaining this to a kid or send me a video step by step on what to do to get this installed and up and running? Or i can give you remote access to my laptop to set this up becuz i genuinely cannot figure this out and its so frustrating

**Comment by Batlez** [comment]
> @Batlez Hi! Thank you for posting this VM. I'm a major noob to the tech world and having trouble with literally everything from start to finish, are you able to pls post a video as if ur explaining this to a kid or send me a video step by step on what to do to get this installed and up and running? Or i can give you remote access to my laptop to set this up becuz i genuinely cannot figure this out and its so frustrating

Please add me on discord: Croakq
